### DISCUSSION TITLE
Venomancer too underpowered! he's useless! Here's an idea.

### DISCUSSION THREAD

**Discussion by Aikirob** [ValveSoftware/Dota2-Gameplay#2873] [Ideas]
Can we make his ulti lethal also make his ulti a bit better, its very old and dated and basically does nothing throughout the entire match. maybe make it a ground aoe that slows in a super large area? with plague wards that pop up with it for its duration? so you get great vision on the map, a good slow and good damage over time? I'm talking tidehunter's ravage kind of aoe.

would make his ulti valuable in a team fight, but not straight up devastating, like a bkb could still rock your shit, but people without it would get fucked, would also be pretty great against people trying to invis or blink away from a team fight.

would also be cool if you had a gem in your item bar, that the plague wards revealed invisible targets, something a little different and special for him.

Also giving the wards venomancers attack modifiers as a talent would be pretty cool.

**Comment by mrstealyommr** [comment]
His whole shtick is jump in ult and force bkbs or suffer huge dmg. Then have your team roll in when theyre half hp or trying to tp to fountain. Hes a terrible hero if you intend to fight straight away.

**Comment by DamionLedner** [comment]
The proposed rework of Venomancer's ultimate is reasonable and necessary in the current [Escape Tsunami for Brainrots](https://escapetsunamiforbrainrots.io) landscape. Transforming Poison Nova into a wide-area ground AoE, with slow, damage over time, and temporary Plague Wards, would make this ultimate truly valuable in combat, true to Venomancer's area control nature, instead of just being a fleeting ability.
